docs/content/guide/directive.ngdoc

```diff
@@ -587,14 +587,15 @@ want to reuse throughout your app.
 In this example we will build a directive that displays the current time.
 Once a second, it updates the DOM to reflect the current time.
 
-Directives that want to modify the DOM typically use the `link` option.
-`link` takes a function with the following signature, `function link(scope, element, attrs) { ... }`
-where:
+Directives that want to modify the DOM typically use the `link` option, which is documented on the {@link ng.$compile `$compile` API} page.
+`link` takes a function with the following signature, `function link(scope, element, attrs, controller, transcludeFn) { ... }`, where:
 
 * `scope` is an Angular scope object.
 * `element` is the jqLite-wrapped element that this directive matches.
 * `attrs` is a hash object with key-value pairs of normalized attribute names and their
   corresponding attribute values.
+* `controller` is the controller for the directive, if defined.
```

Comment:
This is not 100% accurate. Based on the `$compile` API docs, I would suggest something like this:

`controller` is the directive's required controller instance(s) or it's own controller (if any). The exact value depends on the directive's `require` property:

* no controller(s) required: the directive's own controller, or undefined if it doesn't have one
* string: the controller instance
* array: array of controller instances